clio-client/src/main/scala/org/broadinstitute/clio/client/util/IoUtil.scala

```diff
@@ -56,12 +56,33 @@ trait IoUtil {
     gs.exists(path) == 0
   }
 
+  def getMd5HashOfGoogleObject(path: String): String = {
+    new GsUtil(None).md5Hash(path)
+  }
+
+  def getSizeOfGoogleObject(path: String): Long = {
+    new GsUtil(None).du(path).head.split(" +").head.toLong
+  }
+
+  def listGoogleObjects(path: String): Seq[String] = {
+    new GsUtil(None).ls(path)
+  }
+
+
 }
 object IoUtil extends IoUtil {}
```

Comment:
What does this definition do?

Reply:
I'm not exactly sure, but the code doesn't compile when I remove the trait and put all the defs in the object. Gonna leave it as it is.